subscriptions/tasks.py

```diff
@@ -308,9 +343,57 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                                                 kwargs, einfo)
 
 
+class SendNextMessage(BaseSendMessage):
+
+    """
+    Task to load and contruct message and send them off
+    """
+    name = "subscriptions.tasks.send_next_message"
+
+
 send_next_message = SendNextMessage()
 
 
+class SendCurrentMessage(BaseSendMessage):
+
+    """
+    Task to load and contruct last sent message and send it again
+    """
+    name = "subscriptions.tasks.send_current_message"
+
+    def run(self, subscription_id, **kwargs):
+        self.resend_id = kwargs['resend_id']
```

Comment:
I think this should rather be an arg than a kwarg, since it's required, ie. This task will break if we don't supply it.